driver-core/src/main/com/mongodb/client/model/Accumulators.java

```diff
@@ -67,6 +67,51 @@ public static <TExpression> BsonField avg(final String fieldName, final TExpress
         return accumulatorOperator("$avg", fieldName, expression);
     }
 
+    /**
+     * Returns a combination of a computed field and an accumulator that generates a BSON {@link org.bson.BsonType#ARRAY Array}
+     * containing computed values from the given {@code inExpression} based on the provided {@code pExpression}, which represents an array
+     * of percentiles of interest within a group, where each element is a numeric value between 0.0 and 1.0 (inclusive).
+     *
+     * @param fieldName      The field computed by the accumulator.
+     * @param inExpression   The input expression.
+     * @param pExpression    The expression representing a percentiles of interest.
+     * @param method         The method to be used for computing the percentiles.
+     * @param <InExpression> The type of the input expression.
+     * @param <PExpression>  The type of the percentile expression.
+     * @return The requested {@link BsonField}.
+     * @mongodb.driver.manual reference/operator/aggregation/percentile/ $percentile
+     * @since 4.10
+     * @mongodb.server.release 7.0
+     */
+    public static <InExpression, PExpression> BsonField percentile(final String fieldName, final InExpression inExpression,
+                                                                   final PExpression pExpression, final QuantileMethod method) {
+        notNull("fieldName", fieldName);
+        notNull("inExpression", inExpression);
+        notNull("pExpression", inExpression);
+        notNull("method", method);
+        return quantileAccumulator("$percentile", fieldName, inExpression, pExpression, method);
+    }
+
+    /**
+     * Returns a combination of a computed field and an accumulator that generates a BSON {@link org.bson.BsonType#DOUBLE Double }
+     * representing the median value computed from the given {@code inExpression} within a group.
+     *
+     * @param fieldName The field computed by the accumulator.
+     * @param inExpression The input expression.
+     * @param method The method to be used for computing the median.
+     * @param <InExpression> The type of the input expression.
+     * @return The requested {@link BsonField}.
+     * @mongodb.driver.manual reference/operator/aggregation/median/ $median
+     * @since 4.10
+     * @mongodb.server.release 7.0
+     */
+    public static <InExpression> BsonField median(final String fieldName, final InExpression inExpression,  final QuantileMethod method) {
```

Comment:
The parameter name 'InExpression' is used in five other methods as well. This naming choice was made to maintain consistency within the documentation. The usage of the type parameter name `'* @param <InExpression> The type of the input expression.'` follows a consistent approach throughout the entire class. If we want to change this approach here, it would require renaming it in all instances. What are your thoughts on whether we should change it for the entire class?

Reply:
Fine to leave